packages/web/src/components/background-animations/BackgroundWaves.jsx

```diff
@@ -1,138 +0,0 @@
-import { useRef, useEffect, useCallback } from 'react'
```

Comment:
I miss the pretty waves already 🌊 
(but also you could barely see them and they aren't worth the performance costs 🙃 )